rpcs3/Emu/RSX/VK/vkutils/sampler.cpp

```diff
@@ -4,9 +4,46 @@
 
 namespace vk
 {
+	extern VkBorderColor get_border_color(u32);
+
+	static VkBorderColor get_closest_border_color_enum(const color4f& color4)
+	{
+		if ((color4.r + color4.g + color4.b) > 1.35f)
+		{
+			//If color elements are brighter than roughly 0.5 average, use white border
```

Comment:
```suggestion
			// If color elements are brighter than roughly 0.5 average, use white border
```

als0, why 0.5 ? it's 1.35 above

Reply:
It's about 0.45, though the target is 0.5 with a factor. I must have changed it to fix some game many years ago and forgot to update the comment.